client/db/bolt/upgrades.go

```diff
@@ -118,7 +127,8 @@ func v1Upgrade(dbtx *bbolt.Tx) error {
 		return fmt.Errorf("appBucket not found")
 	}
 	skipCancels := true // cancel matches don't get revoked, only cancel orders
-	return reloadMatchProofs(dbtx, skipCancels)
+	matchesBucket := []byte("matches")
+	return reloadMatchProofs(dbtx, skipCancels, matchesBucket)
```

Comment:
`archivedMatchesBucket`? Or would that be confusing?

Reply:
Just trying to make these older upgrades more robust to future changes to package level variables by defining the bucket/key names as they are known to be at that particular version.